### DISCUSSION TITLE
Viewing Point Cloud Model and Dwg / Other cad file format in iTwin viewer

### DISCUSSION THREAD

**Discussion by ajaycctech** [iTwin/itwinjs-core#5842] [General]
Hello, I am new to the Bentley ITwin services.

What are doing : We are creating a web application from where user can upload different cad file format ( dwg, point cloud models , others ) and we are going show that model or files in Bentley iTwin viewer

Challenge : We couldn't find any information around how to view point cloud model into the iTwin viewer. Also directly viewing dwg file format in iTwin viewer is also not supported.

Questions : 
1. Do we need to convert cad file formats to dgn file format before viewing it in iTwin viewer ?
2. Are there any apis available to do this conversions ?
3. If there's no apis, do we have to do this conversion with Microstation ? does microstation supports conversion to dgn file format for all cad file formats ?

**Comment by pratikshan85** [comment]
Yes, the native files can't be directly viewed by iTwin Viewer. You need to create iModel to view the design data in the viewer.
There are several ways to sync to an iModel using input design data (files).

I encourage you to complete an introductory accreditation course that will explain the iModel ecosystem.

Associate level courses

https://developer.bentley.com/accreditation/

Please let me know if you have further queries.
